code/online_challenges/src/Hacker_Rank/3D_Surface_Area/3D_Surface_Area.cpp

```diff
@@ -0,0 +1,66 @@
+
+#include <iostream>
+
+int main()
+{
+int h, w;
+    std::cin >> h >> w;
+    int a[h][w];
```

Comment:
C++ does not support VLAs without compiler extensions; use a `std::vector` instead.